arangograph/cloud-migration-tool.md

```diff
@@ -245,6 +245,31 @@ error code `11` (ERROR_FORBIDDEN).
 The `--source.server-mode` option allows you to specify the desired server mode.
 Allowed values are `readonly` or `default`.
 
+### Supported environment variables
+
+The `arangosync-migration` tool supports the following environment variables:
+
+- `$OASIS_API_KEY`
+- `$OASIS_API_SECRET`
+- `$OASIS_DEPLOYMENT_ID`
+
+Using these environment variables is highly recommended to ensure a secure way
+of providing sensitive data to the application.
+
+### Restrictions and limitations
+
+When running the migration, ensure that your target deployment has the same (or
+bigger) amount of resources (CPU, RAM) than your cluster. Otherwise, the
+migration process might get stuck or require manual intervention. This is closely
+connected to the type of data you have and how it is distributed between shards
+and collections.
+
+In general, the most important parameters are:
+- Total number of leader shards
+- The amount of data in bytes per collection
+
+Both parameters can be retrieved from the ArangoDB Web Interface.
```

Comment:
The main idea of updating migration agent documentation was to have a specific numbers of what we tested - to provide expectations for customers on how much data he can migrate.

From [draft](https://github.com/arangodb/arangosync#restrictions-and-limitations):
```
The largest dataset we tested had 5TB of data and 3800 leader shards. Some collections were as big as 250GB of data.

We expect the migration to work fine even with bigger datasets, but if you have use-case with bigger numbers please reach out to us, so we will perform more tests with datasets similar to yours.
```

@joerg84 please let us know if this is not needed.

Reply:
I would not set the expectation that we'd definitely perform more tests for anyone that gets in contact with us with a larger dataset. 
Agree with the rest of your suggestion @nikita-vanyasin.

Reply:
done.